### ISSUE TITLE
Problem installing plv8 on RHEL 7

### ISSUE THREAD

**Issue by hmarzooq** [plv8/plv8js#479]
Hi.

I'm trying to create plv8-2.3.15 on RHEL 7/x86_64 machine, but getting below error:

```
postgres=# CREATE EXTENSION plv8;
ERROR:  could not load library "/opt/rh/rh-postgresql13/root/usr/lib64/pgsql/plv8-2.3.15.so": /lib64/libc.so.6: version `GLIBC_2.18' not found (required by /usr/local/llvm-lib/lib/x86_64-unknown-linux-gnu/libc++abi.so.1)
postgres=#]#
```
**more info:**
```
 # ll /usr/lib64/libc.so.6  
 lrwxrwxrwx. 1 root root 12 Dec 17 00:05 /usr/lib64/libc.so.6 -> libc-2.17.so

# ll /usr/local/llvm-lib/lib/x86_64-unknown-linux-gnu/libc++abi.so.1
-rwxr-xr-x. 1 root root 358936 Dec 30 04:54 /usr/local/llvm-lib/lib/x86_64-unknown-linux-gnu/libc++abi.so.1

# gcc --version
gcc (GCC) 11.2.1 20210728 (Red Hat 11.2.1-1)
# python --version
Python 2.7.5
# python3 --version
Python 3.6.8

# strings /usr/lib64/libc.so.6 | grep GLIBC_
GLIBC_2.2.5
GLIBC_2.2.6
GLIBC_2.3
GLIBC_2.3.2
GLIBC_2.3.3
GLIBC_2.3.4
GLIBC_2.4
GLIBC_2.5
GLIBC_2.6
GLIBC_2.7
GLIBC_2.8
GLIBC_2.9
GLIBC_2.10
GLIBC_2.11
GLIBC_2.12
GLIBC_2.13
GLIBC_2.14
GLIBC_2.15
GLIBC_2.16
GLIBC_2.17
GLIBC_PRIVATE
```

Any help would be highly appreciated.

Thanks!

**Comment by JerrySievert** [comment]
unfortunately, 2.18 is a v8 requirement.  you have a few options:

1. update your OS to something that ships 2.18
2. update your local copy of libc by either building from source or installing from an alternate [rpm repo](https://pkgs.org/download/libc.so.6(GLIBC_2.18)(64bit)) (for instance)
3. using an older version of plv8 that includes an older version of v8

number 2 might be a worthwhile approach, but I don't run RHEL/CentOS so do not know how it will affect whatever else you are running on that host.

**Comment by hmarzooq** [comment]
Hi Jerry, 

I've tried again to build `libcxx and libcxxabi` from source and plv8-3.0.0 from source as well on RHEL 7.9. I'm getting a different error below

```
# CREATE EXTENSION plv8;
ERROR:  could not load library "/opt/rh/rh-postgresql13/root/usr/lib64/pgsql/plv8-3.0.0.so": /lib64/libstdc++.so.6: version `GLIBCXX_3.4.20' not found (required by /opt/rh/rh-postgresql13/root/usr/lib64/pgsql/plv8-3.0.0.so)
postgres=#
```

I see the systems `/usr/lib64/libstdc++` is pointing to `libstdc++.so.6.0.19` which is of course not **GLIBCXX_3.4.20**. however, I see there is another version `libstdc++.so.6.0.28` available after plv8 build`plv8-3.0.0/build/v8/build/linux/debian_sid_i386-sysroot/usr/lib/i386-linux-gnu/libstdc++.so.6.0.28`

I tried doing `export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/root/plv8-3.0.0-FRESH/build/v8/build/linux/debian_sid_i386-sysroot/usr/lib/i386-linux-gnu/libstdc++.so.6.0.28` but got same error. 


Do you think symlinking the system's `libstdc++.so.6 --> libstdc++.so.6.0.28` would work ?

some info:
```
$ll /usr/lib64/libstdc++.so.6
lrwxrwxrwx. 1 root root 19 Dec 17 00:05 /usr/lib64/libstdc++.so.6 -> libstdc++.so.6.0.19
$ ll /usr/lib64/libstdc++.so.6.0.19
-rwxr-xr-x. 1 root root 995840 Apr  4 20:24 /usr/lib64/libstdc++.so.6.0.19
$ ll /root/plv8-3.0.0-FRESH/build/v8/build/linux/debian_sid_i386-sysroot/usr/lib/i386-linux-gnu/libstdc++.so.6.0.28
-rw-r--r--. 1 376730 89939 1906628 Nov 30  2019 /root/plv8-3.0.0-FRESH/build/v8/build/linux/debian_sid_i386-sysroot/usr/lib/i386-linux-gnu/libstdc++.so.6.0.28
```

**Comment by JerrySievert** [comment]
> version `GLIBCXX_3.4.20' not found 

it is telling you that it cannot find glibc++ v3.4.20, not that libstdc++ is the wrong version.

maybe look to something like this? https://stackoverflow.com/questions/68231387/starting-vs-code-hangs-with-glibcxx-3-4-20-not-found-can-i-point-to-a-libstdc

rhel 7 does not have packaged for 3.4.20
